susemanager-utils/susemanager-sls/src/beacons/pkgset.py

```diff
@@ -1,45 +1,51 @@
 # -*- coding: utf-8 -*-
-'''
+"""
 Watch RPM or DPkg database via cookies and fire
 an event to the SUSE Manager if that has been changed.
 
 Author: Bo Maryniuk <bo@suse.de>
-'''
+"""
 
 from __future__ import absolute_import
-from salt.utils.odict import OrderedDict
 import os
-import logging
-log = logging.getLogger(__name__)
 
+import salt.cache
+import salt.config
 
-__virtualname__ = 'pkgset'
 
-PKG_PLUGINS = OrderedDict([
-        ("/usr/lib/zypp/plugins/commit/venv-zyppnotify", "/var/cache/venv-salt-minion/rpmdb.cookie"),
-        ("/usr/share/yum-plugins/venv-yumnotify.py",     "/var/cache/venv-salt-minion/rpmdb.cookie"),
-        ("/usr/bin/venv-dpkgnotify",                     "/var/cache/venv-salt-minion/dpkg.cookie"),
-        ("/usr/lib/zypp/plugins/commit/zyppnotify",      "/var/cache/salt/minion/rpmdb.cookie"),
-        ("/usr/share/yum-plugins/yumnotify.py",          "/var/cache/salt/minion/rpmdb.cookie"),
-        ("/usr/bin/dpkgnotify",                          "/var/cache/salt/minion/dpkg.cookie")
-    ])
-COOKIE_PATH = None
+__virtualname__ = "pkgset"
+
+SALT_CONFIG_DIR = os.environ.get("SALT_CONFIG_DIR")
+
+__opts__ = salt.config.minion_config(
+    os.path.join("/etc/salt" if SALT_CONFIG_DIR is None else SALT_CONFIG_DIR, "minion")
+)
```

Comment:
Small suggestions to remove the conditional expression in `os.path.join`. I haven't yet found a way to run `black` in GitHub PR reviews, it could be that `black` puts everything on a single line :black_heart: 
```suggestion
SALT_CONFIG_DIR = os.environ.get("SALT_CONFIG_DIR", "/etc/salt")

__opts__ = salt.config.minion_config(
    os.path.join(SALT_CONFIG_DIR, "minion")
)
```

Reply:
Agree, it's better :+1: